.github/workflows/ci.yaml

```diff
@@ -161,28 +161,45 @@ jobs:
       #      if: runner.os == 'Linux'
       #      working-directory: ./editors/code
 
+      # You may fix the code automatically by running `npm run lint:fix` if this steps fails.
       - run: npm run lint
         working-directory: ./editors/code
         if: needs.changes.outputs.typescript == 'true'
 
+      # To fix this steps, please run `npm run format`.
```

Comment:
Why is this twice? One here...

Reply:
Ugh, that was my mistake. I pushed  new patch.